src/store/state/state-model.ts

```diff
@@ -9,8 +9,8 @@ export const stateModel: StateModelIF = {
     entityType: null,
     currentDate: '',
     filingDate: '',
-    filingId: null,
-    correctedFilingId: null,
+    filingId: null as number,
+    correctedFilingId: null as number,
```

Comment:
This is not required but it's explicit.
A better way might be to initialize these to NaN (but that might require other code changes).